gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/io/gravitee/apim/core/log/usecase/SearchConnectionLogUsecaseTest.java

```diff
@@ -182,7 +183,7 @@ void should_return_api_connection_logs_with_only_plan_id_if_plan_cannot_be_found
 
         var result = usecase.execute(
             GraviteeContext.getExecutionContext(),
-            new Input(API_ID, new SearchLogsFilters(FIRST_FEBRUARY_2020, SECOND_FEBRUARY_2020, null))
+            new Input(API_ID, new SearchLogsFilters(FIRST_FEBRUARY_2020, SECOND_FEBRUARY_2020, null, null))
```

Comment:
Maybe we could think to add a `@Builder` annotation to `SearchLogsFilters` record to avoid those `null` parameters 🤔

Reply:
done